cmd/utils.go

```diff
@@ -160,6 +161,24 @@ var globalProfiler interface {
 	Stop()
 }
 
+// Prints a given message after a requested delay.
+func delayPrint(message string, delay time.Duration) chan bool {
```

Comment:
By the time the print initiated, there is a change of lock acquisition.  This doesn't give much value.  We just need to print before lock attempt.  That's good enough.